docs/statistics.md

```diff
@@ -0,0 +1,292 @@
+SRT provides a powerful set of statistical data on a socket. This data can be used to keep an eye on socket's health, and track some faulty situations as well.
+
+Statistics is calculated on each side (receiver and sender) independently, and is not exchanged between peers, unless explicitly stated.
+
+## Total accumulated measurements
+
+### msTimeStamp
+
+Time elapsed since the SRT socket is started (after successful call to `srt_connect(...)` or `srt_bind(...)`), in milliseconds.
```

Comment:
Time elapsed since the SRT socket was started (after successful call to `srt_connect(...)` or `srt_bind(...)`), in milliseconds.